internal/validator/validator.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021-2023 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package validator
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/bufbuild/protovalidate-go"
+	"google.golang.org/protobuf/proto"
+
+	policyv1 "github.com/cerbos/cerbos/api/genpb/cerbos/policy/v1"
+)
+
+var Validator *protovalidate.Validator
+
+func init() {
+	var err error
+	if Validator, err = Init(); err != nil {
+		log.Fatal(err.Error())
+	}
```

Comment:
This is an SDK so we shouldn't just crash the whole program like this. 

Either we should initialise a validator when a client is created and return the error at that point or we should change the `Validate` function to always return false if the validator variable is nil.